sematic/resolvers/cloud_resolver.py

```diff
@@ -211,7 +212,7 @@ def _wait_for_any_remote_job(self) -> Optional[str]:
             ] = api_client.update_run_future_states(
                 list(scheduled_futures_by_id.keys())
             )
-            logger.info("Checking for updates on %s", scheduled_futures_by_id.keys())
+            logger.debug("Checking for updates on %s", scheduled_futures_by_id.keys())
```

Comment:
Users never see these logs anyway, unless they are running non-detached (which they never do). And it is valuable info for us to have access to if something fails weirdly